src/Maths.hs

```diff
@@ -29,7 +29,7 @@ cramer ((v1x, v1y), (v2x, v2y)) (rx, ry) =
       where
         det = determinant ((v1x, v1y), (v2x, v2y))
 
--- | return the intersection point between 2 vectors, if exists
+-- | return, if exist, the intersection point between 2 vectors construct each one with a point and a vector
```

Comment:
Return, if it exists, the intersection point between 2 *segments* defined by a starting point and a direction vector.

Reply:
Also: line too long.